packages/react/src/hooks/participant-hooks.ts

```diff
@@ -35,13 +35,14 @@ export interface UseParticipantsOptions {
  * The useParticipants hook returns all participants (local and remote) of the current room.
  */
 export const useParticipants = (options: UseParticipantsOptions = {}) => {
-  const { updateOnlyOn } = options;
-  const remoteParticipants = useRemoteParticipants({
-    updateOnlyOn,
-  });
+  const [cachedOptions] = React.useState(options);
+  const remoteParticipants = useRemoteParticipants(cachedOptions);
   const { localParticipant } = useLocalParticipant();
 
-  return [localParticipant, ...remoteParticipants];
+  return React.useMemo(
+    () => [localParticipant, ...remoteParticipants],
+    [localParticipant, remoteParticipants],
+  );
```

Comment:
I think wrapping this in a `useMemo` is not necessary. Testing it locally didn't have any effect for me. Different for you?

Reply:
left over from debugging, fixed